### PR TITLE
add og metadata for groups

### PR DESCRIPTION
**Pull request by majakomel**
which makes onebox for groups possible: https://meta.discourse.org/t/onebox-for-groups/79155

### CODE REVIEW

**Comment by SamSaffron** [comment]
awesome, can you add a few screenshots on the topic on meta?

File: spec/controllers/groups_controller_spec.rb [hunk 1]
```diff
@@ -17,6 +17,12 @@
       expect(::JSON.parse(response.body)['basic_group']['id']).to eq(group.id)
     end
 
+    it "responds with HTML" do
+      Guardian.any_instance.expects(:can_see?).with(group).returns(true)
```
Comment (SamSaffron) [inline spec/controllers/groups_controller_spec.rb]: hmmm why is this needed? just create a visible group. the mocking here just risks instability long term

File: app/views/groups/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<% content_for :head do %>
+  <%= raw crawlable_meta_data(title: @title, description: @description_meta) %>
+<% end %>
```
Comment (SamSaffron) [inline app/views/groups/show.html.erb]: we should add one test or 2 that renders views and makes sure content is there.

**Comment by tgxworld** [comment]
Merged in https://github.com/discourse/discourse/commit/76a2fc3d0782aede35b77bae1604ef297d7ee29c

Thanks @majakomel
